src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js

```diff
@@ -119,6 +161,26 @@ export default class Selection extends React.Component<Props> {
         return type;
     }
 
+    buildRequestOptions(
+        requestParameters: Array<SchemaOption>,
+        resourceStorePropertiesToRequest: Array<SchemaOption>,
+        formInspector: FormInspector
+    ) {
+        const requestOptions = {};
+
+        requestParameters.forEach((parameter) => {
+            requestOptions[String(parameter.name)] = parameter.value;
```

Comment:
Why is the `String` call necessary here? Have never seen that before in our codebase :thinking:

Reply:
because the `SchemaOption` type is defined as:
```
export type SchemaOption = {
    infoText?: string,
    name?: string | number,
    title?: string,
    value?: ?string | number | boolean | Array<SchemaOption>,
};
```

Therefore `parameter.name` could be `undefined` in theory and flow complains that i cannot use `undefined` as kay for the assignment. do you have a better idea how to solve this? should we throw an error?

Reply:
I think the `SchemaOption` type is wrong... In a template XML you have to set a name, but the value can be left empty. Therefore I would say that we change the type here.